### PR TITLE
HOTFIX: v1.1.1

### PR DESCRIPTION
**Pull request by MateusStano**
## Fixed

https://github.com/RocketPy-Team/RocketPy/pull/479:
- Opposite static margin bug, which affected all rockets defined using a nose_to_tail coordinate system, causing positive static margins to become negative and negative ones to become positive. Note: this bug did not lead to any issues in terms of the trajectory simulation itself.

https://github.com/RocketPy-Team/RocketPy/pull/478:
- Fixed Bug in 2D .csv-Based Functions for Proper Interpolation Handling and Added Relevant Test Coverage

https://github.com/RocketPy-Team/RocketPy/pull/476:
- Corrected Function.compose extrapolation.
- Added overfill validation in MassBasedTank.fluid_volume.
- Introduced assertion in test_mass_based_tank to prevent future errors in fluid_volume.

### CODE REVIEW

**Review by Gui-FernandesBR — CHANGES_REQUESTED** [review]
Thank you @MateusStano !!

2 changes that we still need to do before merging:
- Update the CHANGELOG file.
- Re-run the SEBLM notebook to update the static margin plot. See:

image

**Comment by Gui-FernandesBR** [comment]
We should have more tests for the tanks, not all the lines are being covered right now, as reported by the codecov

**Comment by Gui-FernandesBR** [comment]
This one solves #482

**Review by Gui-FernandesBR — APPROVED** [review]
Great one!

File: tests/test_tank.py [hunk 1]
```diff
@@ -190,6 +189,10 @@ def test_mass_based_tank():
         discretize=None,
     )
 
+    # Assert volume bounds
+    assert (real_tank_lox.gas_height <= real_tank_lox.geometry.total_volume).all
```
Comment (phmbressan) [inline tests/test_tank.py:193]: This line is wrong, it should be 


```suggestion
    assert (real_tank_lox.gas_height <= real_tank_lox.geometry.top).all
```

**Review by phmbressan — APPROVED** [review]
Great fixes. I just noticed a small assertion mistake and will commit a fix.

**Pull request merged**